.travis.yml

```
@@ -7,8 +7,8 @@ env:
     - SEGFAULT_SIGNALS=all
     - SALT_STATE=scap.scan
   matrix:
-    - OS_VERSION=7 SALT_PILLARROOT=$TRAVIS_BUILD_DIR/tests/pillar/test-scc
-    - OS_VERSION=7 SALT_PILLARROOT=$TRAVIS_BUILD_DIR/tests/pillar/test-oscap
+    - OS_VERSION=7 SALT_PILLARROOT=$TRAVIS_BUILD_DIR/tests/pillar/test-scc SALT_VERSION=3004
+    - OS_VERSION=7 SALT_PILLARROOT=$TRAVIS_BUILD_DIR/tests/pillar/test-oscap SALT_VERSION=3004
```

Same.  Move `SALT_VERSION` to `global:` section.

```suggestion
    - OS_VERSION=7 SALT_PILLARROOT=$TRAVIS_BUILD_DIR/tests/pillar/test-oscap
```